README.md

```diff
@@ -217,6 +217,13 @@ Default: `0`
 Specifies the number of lines at the beginning of a data file that the parser should
 skip over, prior to parsing headers.
 
+##### maxRowSize
+
+Type: `Number`<br>
+Default: `0`
```

Comment:
I'd like to go a different route here. (see below) Please update this to "Default: `Number.MAX_SAFE_INTEGER`"